infoschema/metricschema/table.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metricschema
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/parser/mysql"
+	"github.com/pingcap/tidb/sessionctx"
+)
+
+const (
+	promQLQuantileKey       = "$QUANTILE"
+	promQLLabelConditionKey = "$LABEL_CONDITION"
+	promQRangeDurationKey   = "$RANGE_DURATION"
+)
+
+// MetricTableDef is the metric table define.
+type MetricTableDef struct {
+	PromQL   string
+	Labels   []string
+	Quantile float64
+}
+
+// TODO: read from system table.
+var metricTableMap = map[string]MetricTableDef{
+	"query_duration": {
+		PromQL:   `histogram_quantile($QUANTILE, sum(rate(tidb_server_handle_query_duration_seconds_bucket{$LABEL_CONDITION}[$RANGE_DURATION])) by (le))`,
+		Labels:   []string{"instance", "sql_type"},
+		Quantile: 0.90,
+	},
+	"up": {
```

Comment:
```suggestion
	"uptime": {
```

Reply:
The metric value is not a time, just a bool value( 0 or 1), indicate the instance whether up in the time.